gui/src/pages/AddNewModel/configs/providers.ts

```diff
@@ -56,6 +57,54 @@ export const apiBaseInput: InputDescriptor = {
   required: false,
 };
 
+export const openRouterProvider: ProviderInfo = {
+  title: "OpenRouter API",
+  provider: "openrouter" as ModelProvider,
+  description:
+    "A platform that provides a unified API for accessing various large language models (LLMs) from different providers.",
+  icon: "open-router.png",
+  longDescription: `OpenRouter aims to simplify the integration and management of models by offering a single interface, allowing developers to leverage the capabilities of multiple LLMs without dealing with the complexities of each individual API.  Obtain a API key from the [OpenRouter Dashboard](https://openrouter.ai/settings/keys).`,
+  tags: [ModelProviderTags.RequiresApiKey],
+  params: {
+    apiKey: "",
+  },
+  collectInputFor: [
+    {
+      inputType: "password",
+      key: "apiKey",
+      label: "API Key",
+      placeholder: "Enter your OpenRouter API key",
+      required: true,
+    },
+    ...completionParamsInputsConfigs,
+  ],
+  packages: [],
+  apiKeyUrl: "https://openrouter.ai/settings/keys",
+};
+
+export const loadOpenRouterPackages = async () => {
```

Comment:
can this be moved to OpenRouterServices?

Reply:
This was fixed and moved to OpenRouterServices.